doc/basics.md

```diff
@@ -168,6 +197,38 @@ And another one:
   cout << "number: " << v << endl;
 ```
 
+C++17 Support
```

Comment:
Pure curiosity (I have absolutely no skin in the game): why did you move this?

Reply:
Because it came after "Minifying JSON strings without parsing" but was logically meant to follow "Using the Parsed JSON".

I am fixing my own mistake with this change, btw.